snap-overlay/bin/configure-openstack

```diff
@@ -1,172 +1,88 @@
-#!/bin/bash -e
```

Comment:
configure-openstack isn't super relevant to the goal of supporting core18. I updated it to better match what the configure hook does, as it had bit rotted, and I was doing some experiments with installing a snap with a much more minimal configure hook.